dcsazure_AzureSQL_to_AzureSQL_mask_pl/manifest.json

```diff
@@ -1,15 +1,15 @@
 {
   "name": "dcsazure_AzureSQL_to_AzureSQL_mask_pl",
```

Comment:
Waiting for the testing to complete and documenting the test results before seeking reviews.